packages/nameguard-js/.example.env

```diff
@@ -0,0 +1,8 @@
+# Below is a list of the allowed environment variables:
+
+# These URIs are used inside unit tests and examples for looking up on-chain data.
+# Note: The following environment variables are optional.
+# If they are not set, the application will default to using the free public API.
```

Comment:
Yes it works but it's very slow (62 s). It works because process.env will return undefined which is handled correctly by viem's createClient.

Reply:
On CI without the vars set we would get a viem "too many requests" error (I guess I was wrong calling that a timeout).

Reply:
Locally I could not get this "too many requests" reproduced. Maybe it's because the github CI servers share IPs with other services.

Reply:
Note that you can do this:

```typescript
const publicClient = createPublicClient({
      chain: mainnet,
      transport: http(), // undefined by default, right?
    });
```

That's the default viem example.

Reply:
Maybe like this?

```

# These URIs are used inside unit tests and examples for looking up on-chain data.
# Note: The following environment variables are optional.
# If they are not set, the application will default to using the free public API.
# This can cause "too many requests" errors when running tests frequently.
```